### DISCUSSION TITLE
transition for rounded corners

### DISCUSSION THREAD

**Discussion by Chandler-Zhu** [tailwindlabs/tailwindcss#5210] [Help] [answered]
I wanted to add some rounded corners when user hovers over the element, so I use `transition` class but it didn't work, here is my code below 
https://play.tailwindcss.com/PrCPV5r8uP

**Comment by zeaccs** [comment, marked as answer]
The class `transition` is a [shorthand](https://tailwindcss.com/docs/transition-property#class-reference) for
```css
transition-property: background-color, border-color, color, fill, stroke, opacity, box-shadow, transform, filter, backdrop-filter;
transition-timing-function: cubic-bezier(0.4, 0, 0.2, 1);
transition-duration: 150ms;
```

Transitioning *border-radius* required `border-radius` on `transition-property`, thus replacing `transition` to `transition-all` or extending the `transitionProperty` in `tailwind.config.js`.

https://play.tailwindcss.com/5RmKUEVc4q

```js
theme: {
  extend: {
    transitionProperty: { 
      'border-radius': 'border-radius',
    }
  },
},

// usage: transition-border-radius
```
